src/example-button.module/fields.json

```diff
@@ -140,7 +138,7 @@
                 "type": "color",
                 "inherited_value": {
                   "property_value_paths": {
-                    "color": "module.styles.group_background.color.color",
+                    "color": "theme.secondary_color.color",
```

Comment:
This one could make sense. If you are looking to have the background color set to the `secondary_color` by default on hover. So in this case you would be changing the default behavior of the module to include a background-color hover state change.

Reply:
Agreed, this is a good example of usage of this inheritance, and buttons usually at least change opacity on hover, so we can leave this in.

Reply:
Agreed, but this change is not necessary anymore so I'll just close this PR